### PR TITLE
Java: some edits

### PR DESCRIPTION
**Pull request by renejeglinsky**

### CODE REVIEW

File: guides/databases-sqlite.md [hunk 1]
```diff
@@ -83,6 +83,34 @@ The further configuration depends on whether you run SQLite as an [in-memory dat
...
+<div class="impl java">
+
+### Initial Database Schema
+
+Configure the build to create an initial _schema.sql_ file for SQLite using `cds deploy --to sqlite --dry > srv/src/main/resources/schema.sql`.
+
+::: code-group
+```xml [srv/pom.xml]
+<execution>
+	<id>cds</id>
```
Comment (agoerler) [inline guides/databases-sqlite.md:95]: maybe

```suggestion
	<id>cds.build</id>
```

or

```suggestion
	<id>schema.sql</id>
```

or

```suggestion
	<id>generate.db.schema</id>
```

or alike

Comment (renejeglinsky) [reply guides/databases-sqlite.md:95]: I took it from here: https://pages.github.tools.sap/cap/docs/java/persistence-services#initial-database-schema-1

We would need to adapt that then as well, right?

Comment (renejeglinsky) [reply guides/databases-sqlite.md:95]: @agoerler PING

File: guides/databases-sqlite.md [hunk 2]
```diff
@@ -84,6 +84,34 @@ Further configuration depends on whether you run SQLite as an [in-memory databas
...
+		<commands>
+			<command>deploy --to sqlite --dry > srv/src/main/resources/schema.sql</command>
+		</commands>
+	</configuration>
+</execution>
+```
+:::
+
+
+[Learn more about creating an initial database schema](../java/persistence-services/persistence-services#initial-database-schema-1){.learn-more}
```
Comment (renejeglinsky) [inline guides/databases-sqlite.md:110]:
```suggestion
[Learn more about creating an initial database schema](/java/persistence-services/persistence-services#initial-database-schema-1){.learn-more}
```

File: guides/databases-sqlite.md [hunk 3]
```diff
@@ -84,6 +84,34 @@ Further configuration depends on whether you run SQLite as an [in-memory databas
...
+		<commands>
+			<command>deploy --to sqlite --dry > srv/src/main/resources/schema.sql</command>
+		</commands>
+	</configuration>
+</execution>
+```
+:::
+
+
+[Learn more about creating an initial database schema](/java/persistence-services/persistence-services#initial-database-schema-1){.learn-more}
```
Comment (renejeglinsky) [inline guides/databases-sqlite.md:110]:
```suggestion
[Learn more about creating an initial database schema](/java/persistence-services#initial-database-schema-1){.learn-more}
```

**Pull request merged**